app/Controller/EventsController.php

```diff
@@ -2763,8 +2763,17 @@ public function freeTextImport($id) {
 		
 		if ($this->request->is('post')) {
 			App::uses('ComplexTypeTool', 'Tools');
+
+
+            Configure::write('debug', 2);
```

Comment:
;) sure, but this is not yet final. There was a part1 before, this would be part0, and part 2 and maybe part3 will follow. Atm I'm saving after each issue or when leaving the office.. :) 
I spoke w/ Andras several times and I'll let him know when it's in a state when it can be inspected..

was a quick fix for a local need, I'll do this in some local file which is never sent 2 github,:D